sorc/evs_sref_adjust_precip24_time.fd/makefile

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+#  This makefile is to compile ensembel production generator
+#  Author: Binbin Zhou
+#  OS: Intel WCOSS
+#
+
+FC=   ftn -convert big_endian    
+INC= ${G2_INC4}
+BINDIR= ../../exec
```

Comment:
I suggest adding a line to force the creation of $BINDIR in case the directory does not exist yet. If it does not exist, the executable does not get moved and installed in line 19. I suggest the following be added just after line 10.
`mkdir -p $BINDIR`